src/main/java/frc/robot/Robot.java

```diff
@@ -217,18 +252,47 @@ public void autonomousInit() {
     Config.getInstance().reload();
 
     mState = State.SHOOTING;
-    mShootingState = ShootingState.PREPARE_TO_SHOOT;
-    mStorage.preloadBalls(AUTONOMOUS_BALL_COUNT);
+    mShootingState = ShootingState.IDLE;
+    mIntakeState = IntakeState.IDLE;
+    //mStorage.preloadBalls(AUTONOMOUS_BALL_COUNT);
+    mStorage.preloadBalls(3);
```

Comment:
What's wrong with the commented line? Is that preferable?

Reply:
Yes. I wasn't done with this branch when @maxh65000 decided to merge it.